### PR TITLE
fix(assets/dfn-panel.css): remove on save

### PR DESCRIPTION
**Pull request by sidvishnoi**
Fixes https://github.com/w3c/respec/issues/2845

---
<details><summary>Outdated</summary>
**Revert this PR as soon as validator is fixed.**

w3c/css-validator doesn't support custom properties yet (https://github.com/w3c/css-validator/issues/111), so echidna failed to publish.
Setting position of panel via `panel.style.left` is straightforward, but setting `left` for `.dfn-panel::before` (the caret/triangle), JS won't allow. We would have to change the markup otherwise ~- which is something I'm not sure I want to do~.

~God forgive me, for this hack :pray:~
Update: Hack removed, changed markup instead.
</summary>
<details?

### CODE REVIEW

**Comment by sidvishnoi** [comment]
On a second thought, it might not be a bad idea to change the markup instead, :thinking:

File: src/core/dfn-panel.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ export const name = "core/dfn-panel";
 export async function run() {
   const css = await loadStyle();
   document.head.insertBefore(
-    hyperHTML`<style>${css}</style>`,
+    hyperHTML`<style class="removeOnSave">${css}</style>`,
```
Comment (saschanaz) [inline src/core/dfn-panel.js:13]: We want to enable dfn panel on saved documents, no?

Comment (sidvishnoi) [reply src/core/dfn-panel.js:13]: yes, we want to. but not yet. gotta fix echidna breakage first. will add it back with the fix in https://github.com/w3c/respec/pull/2820

Comment (sidvishnoi) [reply src/core/dfn-panel.js:13]: Did you get a change to see other two attempts?
https://github.com/w3c/respec/pull/2846/commits/86e8f5a88d0e7be7b00f192c610516a54d9f4e6f is good if we want a workaround on css-validator, but way too hacky.
https://github.com/w3c/respec/pull/2846/commits/fcc0c9a662e7e1a1b0181e2d287c017332c3b298 is good practice, but I don't like adding markup for decorative changes.

**Comment by sidvishnoi** [comment]
> No builds/ update here wink

Will need to send another PR for builds/ then. Because release script no longer generates w3c-common. What do you prefer - here or a new PR?

**Comment by saschanaz** [comment]
>Because release script no longer generates w3c-common. What do you prefer - here or a new PR?

Updating w3c-common here sounds fine.

**Pull request merged**